packages/office-addin-dev-settings/src/registry.ts

```diff
@@ -121,11 +121,13 @@ export async function deleteValue(key: RegistryKey, value: string): Promise<void
     };
 
     try {
-      key.winreg.valueExists(value, (valueExistsError, exists) => {
+      key.winreg.valueExists(value, (_, exists) => {
         if (exists) {
           key.winreg.remove(value, onError);
```

Comment:
Don't we still need to resolve after calling key.winreg?

Reply:
It actually does. winreg.remove() will call onError() in all cases (even if successful), which then calls reject() for an error or resolve() for success.  I suppose there might be a better name than onError() to convey this. Any suggestions?